fix-point/src/CopyFiles.hs

```diff
@@ -48,5 +48,8 @@ copyAll source target newName = do
     wantedFiles x = not (x == "fix-point" || x == "three-layer.cabal" ||
                          x == ".git")
 
-renameYaml :: Text -> Text -> Text
-renameYaml new s = unlines [T.replace "three-layer" new x | x <- lines s]
+    toRenameFile :: FilePath -> Bool
+    toRenameFile x = x == "package.yaml" || x == "Makefile"
+
+renameTL :: Text -> Text -> Text
+renameTL new s = unlines [T.replace "three-layer" new x | x <- lines s]
```

Comment:
You probably don't need to do `unlines` and `lines` here. If you use `replace` then it's an extra work. And `T.replace` works over whole content already.